django/core/serializers/python.py

```diff
@@ -63,6 +63,8 @@ def handle_fk_field(self, obj, field):
                 value = None
         else:
             value = getattr(obj, field.get_attname())
```

Comment:
OK, sounds good.
But, in
```
    def value_to_string(self, obj):
        return smart_text(self._get_val_from_obj(obj))
```
`self._get_val_from_obj(obj)` can return `None`, so the result will be `None`, and not empty string.
Should it be considered as another issue?